az-digital/az_quickstart
# Closes #2383 Upgrade tag page view.

idk whats up but i cant add a comment on here without getting an awww snap page so i'll leave this here:
First commit to this, only added a new page view and made a display mode
So, I just changed these 2 files. Let me know if more needs to be changed

## Description

Created a new page view under news/taxonomy_term and a new list display mode for it as well

## Related issues
#2383

## How to test


Ensure page view is there and go to /news/some_term should only display relevant ones

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
   - [x ] New feature
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
